index.html

```diff
@@ -83,17 +77,17 @@ <h4>Update Chromebrew packages:</h4>
         <div class="col-12">
           <div class="bs-example bs-example-type">
             <h4>Install Chromebrew (along with Ruby and Git).</h4>
-            <h5>Supported on Chrome OS systems running on an ARM (ARMv7+), legacy 32-bit x86 (without GUI support) or x86-64 processor.</h5>
+            <h5>Supported on Chrome OS systems running on an ARM (ARMv7+) or x86-64 processor.</h5>
           </div>
           <div class="highlight">
-            <pre><code>curl -Ls git.io/vddgY | bash</code></pre>
+            <pre class="can-copy"><code>$ curl -Ls git.io/vddgY | bash</code></pre>
```

Comment:
`$` should be outside the code block to prevent it from being included in the copy?